client/blocks/reader-full-post/index.jsx

```diff
@@ -1,5 +1,6 @@
 import config from '@automattic/calypso-config';
 import { Gridicon } from '@automattic/components';
+import EmbedContainer from '@automattic/components/src/embed-container';
```

Comment:
As above RE us needing to import from the main package without reaching in.
```suggestion
import { EmbedContainer, Gridicon } from '@automattic/components';
```